### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by mdwiltfong** [Jamesllllllllll/Daily-Dashboard#45]
The repo requires a simple description of how development is performed on the repo. As in, why we work in the `development` branch and how to contribute, etc.

**Comment by klunkyfungus** [comment]
For this issue would it suffice to just make a markdown file explaining everything?

**Comment by mdwiltfong** [comment]
@klunkyfungus ,

Yeah! There are actually a couple ways to do this. I would right a README.md at the root of the project, and have the content only address how to spin up the repo, and how to contribute. 

I would also take advantage of GitHub's Wiki feature to write out the roadmap of the project. 

I would prioritize the README.md file

**Comment by mdwiltfong** [comment]
Here's a rough draft of the readme: https://github.com/Jamesllllllllll/Daily-Dashboard/pull/55
